src/mobile/components/ModalWithdrawConfirmation/index.tsx

```diff
@@ -0,0 +1,95 @@
+        onSubmit,
+    } = props;
+
+    const renderHeader = React.useCallback(() => {
+        return (
+            <div className="pg-exchange-modal-submit-header">
+                {formatMessage({ id: 'page.mobile.wallet.withdraw.modal.confirmation' })}
+            </div>
+        );
+    }, [formatMessage]);
+
+    const renderBody = React.useCallback(() => {
+        return (
+            <div className="pg-exchange-modal-submit-body mobile-modal-body__withdraw-confirm">
+                <div className="mobile-modal-body__withdraw-confirm--block">
+                    <span className="mobile-modal-body__withdraw-confirm--light">
+                        {formatMessage({ id: 'page.mobile.wallet.withdraw.modal.confirmation.warning' })}
+                    </span>
+                </div>
+                <div className="mobile-modal-body__withdraw-confirm--block">
+                    <span className="mobile-modal-body__withdraw-confirm--mute">
+                        {formatMessage({ id: 'page.mobile.wallet.withdraw.modal.confirmation.message1' })}
+                    </span>
+                    <span className="mobile-modal-body__withdraw-confirm--light">
+                        {Decimal.format(amount, precision, ',')}  {currency.toUpperCase()}
+                    </span>
+                </div>
+                <div className="mobile-modal-body__withdraw-confirm--block">
+                    <span className="mobile-modal-body__withdraw-confirm--mute">
+                        {formatMessage({ id: 'page.mobile.wallet.withdraw.modal.confirmation.message2' })}
+                    </span>
+                    <span className="mobile-modal-body__withdraw-confirm--light">{rid}</span>
+                </div>
+            </div>
+        );
+    }, [formatMessage, amount, currency, precision, rid]);
+
+    const renderFooter = React.useCallback(() => {
+        return (
+            <div className="pg-exchange-modal-submit-footer modal-footer__withdraw-confirm">
+                <Button
+                    block={true}
+                    className="btn-block mr-1 mt-1 btn-lg"
+                    onClick={onDismiss}
+                    size="lg"
+                    variant="danger"
+                >
+                    {formatMessage({ id: 'page.body.wallets.tabs.withdraw.modal.button.cancel' })}
+                </Button>
+                <Button
+                    block={true}
+                    className="btn-block mr-1 mt-1 btn-lg"
+                    onClick={onSubmit}
+                    size="lg"
+                    variant="primary"
+                >
+                    {formatMessage({ id: 'page.body.wallets.tabs.withdraw.modal.button.withdraw' })}
+                </Button>
+            </div>
+        );
+    }, [formatMessage, onSubmit, onDismiss]);
+
+    return (
+        <Modal title={renderHeader()} onClose={onDismiss} isOpen={props.show}>
+            {renderBody()}
+            {renderFooter()}
```

Comment:
Fine, but usually, functions are used when there is some logic, for example show or not show. In this case there is no any logic, so you can put that tags inside <Modal/> wihout wrapping them in methods. The code will be more clear, IHMO.